plugin-api/src/main/java/org/hyperledger/besu/plugin/services/query/IbftQueryService.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.query;
+
+import org.hyperledger.besu.plugin.data.Address;
+import org.hyperledger.besu.plugin.data.BlockHeader;
+
+import java.util.Collection;
+
+/** Allows for the IBFT specific aspects of the block chain to be queried from plugins. */
```

Comment:
No need to mention plugins.
```suggestion
/** Provides access to IBFT2-specific information about the blockchain. This service is only available when using IBFT2. */
```

Reply:
done